cartaya/inspired by jennifer de walt/2.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="2.css">
+    <title>clickme</title>
+</head>
+<body>
+    <form action="3.html" method="get">
+        <input class="button" type="submit" value="Click me!">
+    </form>

```

Comment:
Always use a convention to the names of all classes. Don’t use the tags name for the class, this is a bad practice